docs/index.md

```diff
@@ -178,19 +238,41 @@ class MyApp : Application() {
 }
 ```
 
+This over-rides `onCreate()` to call `init` on FormulaAndroid. The important thing here is that you have a stream that creates an new
+instance of your `CounterFormula` and calls update passing in the formula as an observable and a reference
+to the `render` function in your Activity.
+
+Next you will need to update your application in your `manifest.xml` to reference your new app class. Your new one 
+should look like this:
+```xml
+  <application
```

Comment:
I would probably avoid introducing such details as most Android devs should know how to register `Application`.

Reply:
I could change this to just mention to remember to update your application to use the new MyApp class.